packages/valory/skills/abstract_round_abci/models.py

```diff
@@ -427,6 +431,58 @@ def __init__(
         kwargs["skill_context"] = skill_context
         super().__init__(*args, **kwargs)
 
+    @property
+    def validator_to_agent(self) -> Dict[str, str]:
```

Comment:
What is the difference between a validator and an agent?

Reply:
Validator is the tendermint part, it is identified with a different key than the agent.
This key is used for consensus in tendermint itself. Because malicious behavior can be comitted in consensus-level as well, we need a way to punish the agent who's validator committed the act.